### PR TITLE
Handle IE9 stylesheet limit

### PR DESCRIPTION
**Pull request by m-mujica**
https://github.com/stealjs/steal/issues/123

bs_win7_ie_9 0
bs_win7_ie_9 0 1

### CODE REVIEW

**Comment by m-mujica** [comment]
The test page in Chrome for comparison.

screen shot 2017-04-12 at 17 36 12

File: test/ie-import-limit/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require("./styles.css");
+
+var yellow = "rgb(255, 255, 0)";
+var s = getComputedStyle(document.body);
```
Comment (m-mujica) [inline test/ie-import-limit/main.js]: remove this line

File: test/ie-import-limit/main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	var s = getComputedStyle(document.body);
+
+	if (window && window.assert) {
+		assert.equal(s.backgroundColor, yellow, "should apply styles correctly");
+		done();
+	}
+	else {
+		console.log(s.backgroundColor);
+	}
+}, 300);
```
Comment (m-mujica) [inline test/ie-import-limit/main.js]: maybe add code to use smaller timeouts in modern browsers.

File: test/ie-import-limit/styles/common32.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: yellow;
+}
+
+#app {
```
Comment (m-mujica) [inline test/ie-import-limit/styles/common32.css]: remove this

**Comment by m-mujica** [comment]
hmm got some tests failures in Firefox and Safari, too. It seems my fixed timeout wasn't doing the trick...

I just pushed a polling based approach to wait for the css or timeout kinda like Funcunit. Hopefully this makes it so modern browsers run tests fast and we get the failure only in IE9

**Comment by m-mujica** [comment]
💥  that's what I'm talking about

google chrome

...time to add the hack

**Comment by m-mujica** [comment]
@matthewp I need to keep some state (ugh! I know) in steal-css, keep the count of stylesheets, the last one created and stuff.

I'm having trouble to get it to work, the module is eval-ed everytime a css file is loaded, adding stuff to the global object does not work (ugh, I know ++).

Thoughts?

**Comment by matthewp** [comment]
The module is evaled every time a css file loads? That doesn't sound right, must be something going on there, right?

**Comment by matthewp** [comment]
👍 LGTM

**Pull request merged**